### ISSUE TITLE
Once other clients support verif over DMs, remove labs flag & always send over DM.

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-web#11369]
Follow-up to https://github.com/vector-im/riot-web/issues/11251

**Comment by jryans** [comment]
Hmm, so this verif over DM feature was merged into the cross-signing labs flag, and there's now only one overall flag.

Does "other clients" mean "other Riot platforms" here? If so, it seems okay, and this can be closed as the flags merged together (it would go out of labs as part of https://github.com/vector-im/riot-web/issues/11219). If we're waiting on non-Riot clients as well, I'd like to know more about that.

@lampholder, can you provide more detail?

**Comment by lampholder** [comment]
IIRC I only meant 'other Riot platforms'

**Comment by jryans** [comment]
Okay, will track as part of https://github.com/vector-im/riot-web/issues/11219 then.
